### PR TITLE
Android Tv Functionality

### PR DESCRIPTION
**Pull request by mul1sh**
This  PR will eventually fix https://github.com/mysteriumnetwork/node/issues/1866 and its currently work in progress.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -121,3 +121,6 @@ iOSInjectionProject/
 
 # Firebase Crashlytics
 GoogleService-Info.plist
+
+#OSX
+.DS_Store
```
Comment (anjmao) [inline .gitignore:126]: Binary file .DS_Store is also pushed to the repo.

File: android/app/build.gradle [hunk 2]
```diff
@@ -166,6 +166,9 @@ dependencies {
     implementation 'network.mysterium:mobile-node:0.22.5'
     // Comment network.mysterium:mobile-node and replace with your local path to use local node build.
     // implementation files('/Users/anjmao/go/src/github.com/mysteriumnetwork/node/build/package/Mysterium.aar')
+
+
+    implementation 'com.google.firebase:firebase-analytics:17.2.2'
```
Comment (anjmao) [inline android/app/build.gradle:171]: Why is this needed?

**Comment by anjmao** [comment]
@mul1sh Nice work! Is it working on tv already? If you need any help feel free to ask here 👍

**Comment by mul1sh** [comment]
@anjmao yep its working on TV, but I need to modify it a bit so that instead of the navigation drawer  for the account settings there is a popup instead  for ease of navigation, but only in the TV. Will also make search more intuitive for TV too.

Will push these commits + fixes of the above in the next few days and of course in case of any queries, I'll let you know 😃

**Comment by chompomonim** [comment]
I have to close this PR as it is outdated.
